April23/update-cycle.md

```diff
@@ -0,0 +1,74 @@
+
+Furthermore, the new update cycle has caused confusion among players who are not familiar with the mods. They may not understand why their favourite mods no longer work, leading to frustration and disappointment.
+
+# Modders' opinions
+
+Modders in the Minecraft modding community have shared their feedback on the impact of Mojang's new update release cycle on modders.
+
+## Positive feedback
+
+Some modders have praised Mojang's new update cycle for providing players with more frequent updates and bug fixes. This, in turn, creates a better gaming experience for players. Additionally, some modders believe that the new update cycle can help modders by providing them with a more stable platform to work with.
+
+One modder states that "Mojang's new schedule is much more flexible in that it allows them to fix bugs and develop features in parallel. No longer will they have to work a few silent months to release a big update all at once, now they can have a more streamlined release process with smaller, more often updates where all experimental content wise changes are offloaded into the experimental datapack instead of only snapshots, which means we can use them sooner."
+
+## Negative feedback
+
+On the other hand, some modders have expressed concern about the impact of the new update cycle on modders. One of the main concerns is that updates may cause mods to break more often, leading to frustration for both modders and players.
+
+Another modder states that "It's also not really giving mods a good period to "settle" which I believe is harmful to mod pack communities and servers while beneficial to vanilla players."
+
+Overall, the impact of Mojang's new update cycle on modders is still up for debate. While some modders have provided positive feedback, others have expressed concerns about the impact on modders.
+# Conclusion
+
+It is clear that Mojang's update cycle has had a significant impact on modders. While some modders have been able to adapt and continue making money, others have struggled to keep up with the constant changes and updates to the game. The lack of an official mod API has also made it difficult for modders to have their work endorsed and distributed through a central repository in-game.
+
+Despite these challenges, Mojang's priority is the game itself. While they are aware of the problems that the update system creates for modders, they are putting little focus on making the workload easier for said creators. This means that modders will need to continue to adapt and find new ways to make their work compatible with the latest updates.
+
+Overall, Minecraft modding's future remains uncertain. While some modders have monetised their work, it is unclear whether this trend will continue. As the game continues to evolve and change, modders will need to find new ways to keep up with the latest updates and changes. Only time will tell how this will impact the Minecraft modding community in the long run.
```

Comment:
```suggestion
The future of Minecraft modding remains uncertain. As the game continues to evolve and change, modders will need to find new ways to keep up with the latest updates and changes. Only time will tell how this will impact the community in the long run.
```